fcomm/tests/proof_tests.rs

```diff
@@ -47,7 +48,7 @@ fn test_eval_expression() {
 
     cmd.assert()
         .success()
-        .stdout("{\"expr\":\"((lambda (a b) (+ (* a 3) b)) 9 7)\",\"env\":\"nil\",\"cont\":\"Outermost\",\"expr_out\":\"34\",\"env_out\":\"nil\",\"cont_out\":\"Terminal\",\"status\":\"Terminal\",\"iterations\":17}");
+        .stdout("{\"expr\":\"((lambda (.lurk.user.a .lurk.user.b) (+ (* .lurk.user.a 3) .lurk.user.b)) 9 7)\",\"env\":\"nil\",\"cont\":\"Outermost\",\"expr_out\":\"34\",\"env_out\":\"nil\",\"cont_out\":\"Terminal\",\"status\":\"Terminal\",\"iterations\":17}");
```

Comment:
Here and below, why is the output being created relative to a different package than the one in which the input was read? In other words, the source refers to `a` — even though this is read as `.lurk.user.a`, my expectation is that it will still be printed as `a`, since the printing should be relative to the same `.lurk.user` package that was current when the symbol was read. What detail am I missing?

Reply:
It's because I didn't change `fcomm` to print it with the state that was used to read.

Reply:
Maybe I don't understand, but this seems to forego the payoff for having a state in the first place. The idea of there existing a current-package is that it should be used for both reading and writing. If that's awkward to provide, maybe that's a sign further design is needed.

Reply:
It's not awkward. The REPL code does this very easily. It's the fcomm code that is not easily suitable for this